### PR TITLE
feat(design-system): add design-system package

### PR DESCRIPTION
**Pull request by elbalexandre**
## Description

- add minimal design system components
- update install-form recipe

## Additional Notes

`NodeNext` resolution module was breaking a drizzle type (`.defaultNow()` that was replaceable by `.default(sql`now()`)`.
I thought it was okay to apply this change, but I realise that react canary imports from next was also broken in the web app using this value for resolution module.

Too bad for three-shaking: I updated the design-system to old fashion src + barrel...

## Checklist:

Before you create this PR, confirm all the requirements listed below by checking the checkboxes like this (`[x]`).

- [X] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have added tests to cover the new feature or fixes.

### CODE REVIEW

File: docs/recipes/instance-install-form.md [hunk 1]
```diff
@@ -33,38 +42,74 @@ export default function InstallPage() {
...
+          <h3>Generate an OAuth Client</h3>
+          <p>In the Security page, you will have to create a new OAuth client.</p>
+          <p>
+            Click on <strong>Generate OAuth Client</strong>.
+          </p>
+          <p>
+            Copy the <strong>Client Id</strong> and the <strong>Client Secret</strong>.
+          </p>
+        </InstructionsStep>
+        <InstructionsStep index={2}>
```
Comment (ValentinH) [inline docs/recipes/instance-install-form.md:67]:
```suggestion
        <InstructionsStep index={3}>
```

**Review by ValentinH — APPROVED** [review]
Looks good!

File: packages/config/next/next.config.js [hunk 2]
```diff
@@ -1,12 +1,13 @@
 /** @type {import('next').NextConfig} */
```
Comment (proquig) [inline packages/config/next/next.config.js:1]: You can remove this line

File: packages/tsconfig/base.json [hunk 3]
```diff
@@ -6,10 +6,10 @@
     "declaration": true,
     "declarationMap": true,
     "esModuleInterop": true,
+    "moduleResolution": "node",
```
Comment (proquig) [inline packages/tsconfig/base.json:9]: If no there is not change here let's keep it as before in alphabetical order

**Review by proquig — APPROVED** [review]

**Pull request merged**